scrapyard/ghz.cpp

```diff
@@ -0,0 +1,40 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
+
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
+
+#include <cudaq.h>
+#include <iostream>
+
+
+int main() {
+
+  auto &platform = cudaq::get_platform();
+  
+  std::string backend = "oqc;emulate;false;url;http://localhost:5000;email;{};password;{}";
```

Comment:
this file is just an example to show the oqc config string if people want to test locally themselves, will remove before merging when squashing commits